.eslintrc

```diff
@@ -1,9 +1,9 @@
 {
 	"root": true,
```

Comment:
@niyobobo , For the sake of the pretty codes, it would be better to work on the indentation in the file.